backend/compact-connect/lambdas/python/common/cc_common/event_bus_client.py

```diff
@@ -156,12 +160,14 @@ def publish_license_encumbrance_lifting_event(
         :param provider_id: The provider ID
         :param jurisdiction: The jurisdiction of the license
         :param license_type_abbreviation: The license type abbreviation
+        :param effective_date: The effective lift date of the encumbrance
         """
         event_detail = {
             'compact': compact,
             'providerId': provider_id,
             'jurisdiction': jurisdiction,
             'licenseTypeAbbreviation': license_type_abbreviation,
+            'effectiveDate': effective_date,
```

Comment:
It's funny - I think I made this same change to the event interfaces as part of my [AA-notification PR](https://github.com/csg-org/CompactConnect/pull/864). We'll have to watch out for merge conflicts / duplication here and to be sure we approach it consistently across the interface.